extensions/wikia/AdEngine/js/run/mercury.run.js

```diff
@@ -81,9 +81,17 @@ require([
 		adInfoListener.run();
 		slotStateMonitor.run();
 		actionHandler.registerMessageListener();
+
+		window.addEventListener('adengine.emitter', function (event) {
+			adEngineBridge.passSlotEvent(event.detail.slotName, event.detail.eventName);
+		});
 	});
 
 	mercuryListener.afterPageWithAdsRender(function () {
 		callBiddersOnConsecutivePageView();
 	});
+
+	mercuryListener.onMenuOpen(function () {
```

Comment:
In https://github.com/Wikia/app/blob/327eda7f4904975cc9db1484e900ddee30077264/extensions/wikia/AdEngine/src/templates/big-fancy-ad-above-mobile-config.js we have access to `adSlot` and `mercuryListener` maybe we could define it there?

It's more obvious for me to have all BFAA configuration in one place.

Reply:
Did you think about removing the callback after the page change?

Reply:
Moved and added. Thanks!